code/debugconsole/console.cpp

```diff
@@ -488,6 +488,22 @@ void dc_putc(char c)
 	line_str->push_back(c);
 }
 
+bool handle_textInputEvent(const SDL_Event& event) {
+	if (event.text.text[0] != '\0' && event.text.text[0] != '\1') {
+		int key_used = 0;
+		for (char c : event.text.text) {
+			if (c < 32)
+				break;
+
+			if ((c != 255) && (dc_command_buf.size() < MAX_CLI_LEN)) {
```

Comment:
Flagged by CI, `(char c != 255)` is apparently being treated as as a numeric int8_t by gcc instead of a character, as its stating that the expression will always be true.  Typecasting or a different check may be needed.

Reply:
I've just removed that check.
Seems to have been a remnant of checking an error code of FSO-internal get keycode methods, so not needed anymore